native/game_backend/src/player.rs

```diff
@@ -121,13 +126,25 @@ impl Player {
 
     pub fn add_cooldown(&mut self, skill_key: &String, cooldown_ms: u64) {
         self.cooldowns.insert(skill_key.to_string(), cooldown_ms);
+        self.available_burst_loads -= 1;
```

Comment:
The available burst loads are something that only affect the basic attacks right? if that's the case maybe we should check that the skill key is the same as the `BASIC_SKILL_KEY` variable

Reply:
Right! I added a validation for this